docs/platform/concepts/managed-users.rst

```diff
@@ -0,0 +1,10 @@
+Managed users 
+==============
+
+The managed users feature provides a centralized way of managing all of your organization's users, including editing their profiles, resetting passwords, and :doc:`setting authentication policies </docs/platform/howto/set-authentication-policies>`.
+
+When you :doc:`verify a domain </docs/platform/howto/manage-domains>`, existing organization users automatically become managed users. New users can be provisioned and managed through your identity provider (IdP). 
```

Comment:
Remove "New users can be provisioned and managed through your identity provider (IdP)." as it doesn't belong to this feature.